packages/effect/test/Duration.test.ts

```diff
@@ -388,7 +388,7 @@ describe("Duration", () => {
 
   it("toSeconds", () => {
     expect(Duration.millis(1).pipe(Duration.toSeconds)).toBe(0.001)
-    expect(Duration.nanos(1n).pipe(Duration.toSeconds)).toBe(9.999999999999999e-10)
+    expect(Duration.nanos(1n).pipe(Duration.toSeconds)).toBe(1e-9)
```

Comment:
This imprecision was unnecessarily caused by the double division we performed before.

I consider this a bug fix.